src/main/scala/fundamentals/level03/ExceptionExercises.scala

```diff
@@ -94,16 +94,19 @@ object ExceptionExercises {
   def createPerson(name: String, age: String): Person = ???
 
   /**
-    * Implement the function createValidPeople that only uses the collect function on List
-    * to create a List of Person instances from personStringPairs. It should not throw any Exceptions.
+    * Implement the function createValidPeople to create a List of Person instances
+    * from personStringPairs. It should not throw any Exceptions.
+    * It should only catch Exceptions thrown by createPerson.
     *
     * scala> createValidPeople
     * = List(Person("Tokyo", 30), Person("Berlin", 43))
     *
+    * Hint: Use `map` and `flatten`
+    *
     * What issues do you run into (if any)?
     */
   def createValidPeople: List[Person] = {
-    personStringPairs.collect {
+    personStringPairs.map {
```

Comment:
I'm worried that this derails it a little and you'll have to explain flattening a List. 

Which is okay I guess. But I'm happy to leave this up to you.